source/retryable-writes/tests/README.rst

```diff
@@ -296,6 +296,23 @@ supported write operations:
   - ``bulkWrite()`` with ``ordered=true`` (no ``UpdateMany`` or ``DeleteMany``)
   - ``bulkWrite()`` with ``ordered=false`` (no ``UpdateMany`` or ``DeleteMany``)
 
+Prose Tests
+===========
+
+The following tests ensure that retryable writes work properly with replica sets
+and sharded clusters.
+
+#. Test that retryable writes raise an exception when using the MMAPv1 storage engine,
+   available in both MongoDB 3.6 and 4.0. For this test, execute a write
+   operation, such as ``insertOne``, which should generate an
+   ``IllegalOperation`` exception. Assert that the error message is the
+   replacement error message::
+
+  This MongoDB deployment does not support retryable writes. Please add
+  retryWrites=false to your connection string.
+
+   and the error code is 20.
```

Comment:
This formatting is off here (https://github.com/mongodb/specifications/blob/7dcc969631d77d01a4a09530f2646748414595b0/source/retryable-writes/tests/README.rst#prose-tests).

Reply:
Fixed